app/controllers/drivers_controller.rb

```diff
@@ -11,7 +11,7 @@ def new
   end
 
   def create
-    driver = Driver.new(driver_params)
+    @driver = Driver.new(driver_params)
```

Comment:
i thought you didnt need to do this?

Reply:
> i thought you didnt need to do this?

Wont the helper that calls `def driver` return `@driver` which in the context of `create` is blank?

Reply:
sure, but i thought you checked if this worked without this?
It makes sense to require it though i think

Reply:
> sure, but i thought you checked if this worked without this?

yep it works even as `driver`, so should i change it back to `driver` instead?

Reply:
yea

Reply:
> It makes sense to require it though i think

Okay suddenly it doesnt work again, 
where should i require the `driver`?

Reply:
just make work haha